config/test.secret.example.exs

```diff
@@ -4,3 +4,11 @@ use Mix.Config
 # config :movie_wager_api, MovieWagerApi.Repo,
 #   username: "postgres",
 #   password: "postgres"
+
+# Replace with your twitter app credentials
```

Comment:
yeah, taking a second pass at this, I realize my comment was incorrect and best ignored :) I think what you have is great and if you ever wanted to remove the "ask Scott for credentials" form the Readme, we could tackle that later on as this project is primarily for two/three people.